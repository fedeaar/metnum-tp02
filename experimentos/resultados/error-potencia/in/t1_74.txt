-5.036549522287351692e+02 -1.669141473636329209e+01 1.834207352260595769e+02 1.329134652893035025e+02 -1.194174637837904562e+00 1.039446286399542316e+02 2.247656975228072795e+01 1.767543914681999695e+01 5.083007337524193048e+01 1.347415814797129769e+02 1.997117780871473869e+01 -5.935299655068901892e-01 5.975501374572551150e+00 1.486032845480752087e+02 1.566580561549577055e+02 -5.343295910888826938e+01 -8.464015232895993890e+00 -4.720242377171579307e+01 -2.010500328831749783e+00 1.687038472577142727e+02
-1.669141473636326722e+01 4.620464912635823680e+02 4.052212819375450081e+00 -1.083898251700844639e+01 -8.574731817593690408e+00 -7.008710069166649248e+01 -1.221774729305459886e+02 -1.628416769499485781e+02 -1.399056857405991821e+02 -5.147880099528130415e+01 -5.632761196607903997e+01 -4.165154433626430119e+01 -1.713292827955683606e+01 1.538067625151405515e+01 -3.368016962872633968e+00 -2.227821718557923134e+02 -7.232357772928406803e+01 -2.360748751650241104e+02 -3.981837213986551660e+01 -5.234068219751009110e+00
1.834207352260596053e+02 4.052212819375450081e+00 -5.927372327982806155e+02 1.637523906914692873e+02 -2.323749359323770691e-01 1.368636408352228955e+02 4.488373770968942722e+01 4.483374810818276046e+01 8.201818400875758641e+01 1.717890691602829349e+02 3.241909139748428004e+01 5.225900084087625963e+00 9.739902821070407057e+00 1.791541474892088104e+02 1.916623587524513823e+02 -3.338206068163069773e+01 2.699207786146295973e-03 -2.387952561158220277e+01 3.234786094523443456e+00 2.066292922789247655e+02
1.329134652893035025e+02 -1.083898251700844462e+01 1.637523906914692873e+02 -5.670108760914727100e+02 -8.625372653278253754e-01 9.452243243319026078e+01 2.298796090387774171e+01 1.966758046531218440e+01 4.873822515372273756e+01 1.215900383075186824e+02 1.918164382390093436e+01 4.617640357338790302e-01 5.745920890000684622e+00 1.323803269957201394e+02 1.400221310063572560e+02 -4.242569764523886988e+01 -5.838427532026527089e+00 -3.654341382418287054e+01 -8.476673766374314134e-01 1.508270220168182618e+02
-1.194174637837903230e+00 -8.574731817593690408e+00 -2.323749359323780128e-01 -8.625372653278250423e-01 6.985707559310036459e+02 -3.779138814269226376e+00 -6.210100881841070120e+00 -8.247769492343085318e+00 -7.170922750260258383e+00 -2.914755664902186183e+00 -2.885919825539913219e+00 -2.097436053971771575e+00 -8.775621180336967697e-01 4.213445021805792834e-01 -5.425860469311724055e-01 -1.109896194166720029e+01 -3.624287087290304576e+00 -1.178361961368026023e+01 -2.001688699949534023e+00 -6.652883713672430588e-01
1.039446286399542174e+02 -7.008710069166649248e+01 1.368636408352228671e+02 9.452243243319023236e+01 -3.779138814269226376e+00 -2.048942148433837929e+02 -2.467426344554307605e+01 -4.195938887071314127e+01 -9.718781829793909921e+00 8.214263159023767003e+01 -4.277507191912996554e+00 -1.451050692412325560e+01 -1.374237349156272270e+00 1.149710131560512849e+02 1.145868624017575570e+02 -1.147388406832331498e+02 -3.068711738424426017e+01 -1.146272540965963458e+02 -1.493789498936761007e+01 1.228547503211815979e+02
2.247656975228073151e+01 -1.221774729305459886e+02 4.488373770968942011e+01 2.298796090387774171e+01 -6.210100881841070120e+00 -2.467426344554306183e+01 2.255748822415618520e+02 -1.086777026925049086e+02 -8.519350525710240163e+01 -4.726930192181082901e+00 -3.441310213448389277e+01 -2.897084088730121820e+01 -1.049000789389264554e+01 4.473879311298029648e+01 3.360722203873244496e+01 -1.664864634226728128e+02 -5.201014943558617176e+01 -1.742594394834343348e+02 -2.802683235808752471e+01 3.507880422508247875e+01
1.767543914682000405e+01 -1.628416769499485781e+02 4.483374810818276046e+01 1.966758046531219151e+01 -8.247769492343085318e+00 -4.195938887071315548e+01 -1.086777026925048801e+02 2.294016410112840276e+02 -1.188297019556542011e+02 -1.774629136083435554e+01 -4.795268124380977781e+01 -3.889761021385436379e+01 -1.460773496158219231e+01 4.758247185274720437e+01 3.193644435508842960e+01 -2.193034983919359320e+02 -6.920577485125070893e+01 -2.302800743241008945e+02 -3.750870956642506826e+01 3.289641686422240241e+01
5.083007337524190916e+01 -1.399056857405992105e+02 8.201818400875758641e+01 4.873822515372273756e+01 -7.170922750260258383e+00 -9.718781829793913474e+00 -8.519350525710237321e+01 -1.188297019556542011e+02 7.823449471023960200e+01 1.797157836730692182e+01 -3.514501860632417873e+01 -3.259347334013576614e+01 -1.073250012226147376e+01 7.584188265984573718e+01 6.474900193817423144e+01 -1.959445413351782008e+02 -5.979159192076845386e+01 -2.035858252808623945e+02 -3.177950475474743186e+01 6.847996448566264860e+01
1.347415814797130054e+02 -5.147880099528130415e+01 1.717890691602829349e+02 1.215900383075186824e+02 -2.914755664902186183e+00 8.214263159023769845e+01 -4.726930192181097112e+00 -1.774629136083435554e+01 1.797157836730694669e+01 -3.072461225714400257e+02 6.775568167840662070e+00 -9.305594858854576046e+00 1.968590200848705951e+00 1.417217212492790281e+02 1.452885422713801802e+02 -9.661083413499120809e+01 -2.308571009157111931e+01 -9.359760020862398733e+01 -1.024091542805382282e+01 1.561224022711316195e+02
1.997117780871474579e+01 -5.632761196607903997e+01 3.241909139748428714e+01 1.918164382390093436e+01 -2.885919825539913219e+00 -4.277507191912996554e+00 -3.441310213448389277e+01 -4.795268124380976360e+01 -3.514501860632417163e+01 6.775568167840670952e+00 1.567663902832314875e+02 -1.313393187187576139e+01 -4.346197484217500850e+00 3.005067122422244097e+01 2.555201810386854078e+01 -7.878522282935806231e+01 -2.406815543820115977e+01 -8.188646705342833343e+01 -1.280096642802727125e+01 2.701390207107818497e+01
-5.935299655068897451e-01 -4.165154433626430119e+01 5.225900084087622410e+00 4.617640357338776980e-01 -2.097436053971771130e+00 -1.451050692412325560e+01 -2.897084088730121820e+01 -3.889761021385437800e+01 -3.259347334013576614e+01 -9.305594858854572493e+00 -1.313393187187576139e+01 5.069323473333300853e+02 -3.997185530755883676e+00 7.154056085905578399e+00 2.815008014308392426e+00 -5.501290217677853178e+01 -1.765358006144679948e+01 -5.807722266234809894e+01 -9.657974108709190375e+00 2.643541898141098301e+00
5.975501374572551150e+00 -1.713292827955683606e+01 9.739902821070407057e+00 5.745920890000686398e+00 -8.775621180336967697e-01 -1.374237349156272270e+00 -1.049000789389264554e+01 -1.460773496158219231e+01 -1.073250012226147376e+01 1.968590200848705063e+00 -4.346197484217500850e+00 -3.997185530755883676e+00 1.736729846512103279e+02 9.043235190227081333e+00 7.668370687738939928e+00 -2.394284068288037659e+01 -7.319781401411468380e+00 -2.489110474117137528e+01 -3.894854859286492665e+00 8.104950072830563457e+00
1.486032845480752087e+02 1.538067625151404272e+01 1.791541474892088104e+02 1.323803269957201110e+02 4.213445021805785062e-01 1.149710131560512565e+02 4.473879311298029648e+01 4.758247185274720437e+01 7.584188265984573718e+01 1.417217212492790566e+02 3.005067122422243386e+01 7.154056085905578399e+00 9.043235190227083109e+00 -7.971003863541044439e+02 1.542823914085291506e+02 -1.103050127502733879e+01 5.131270767400910593e+00 -2.450388263844354597e+00 5.423744672240850662e+00 1.664440052457680963e+02
1.566580561549577055e+02 -3.368016962872674824e+00 1.916623587524513823e+02 1.400221310063572844e+02 -5.425860469311719614e-01 1.145868624017575570e+02 3.360722203873243075e+01 3.193644435508842250e+01 6.474900193817423144e+01 1.452885422713801518e+02 2.555201810386854078e+01 2.815008014308389317e+00 7.668370687738938152e+00 1.542823914085291506e+02 -5.937394482605790245e+02 -3.755152192246828236e+01 -2.892996455867172134e+00 -2.993301493728190721e+01 1.180548135923050523e+00 1.770235011095859363e+02
-5.343295910888827649e+01 -2.227821718557923703e+02 -3.338206068163074036e+01 -4.242569764523888409e+01 -1.109896194166719852e+01 -1.147388406832331498e+02 -1.664864634226728128e+02 -2.193034983919359320e+02 -1.959445413351782292e+02 -9.661083413499126493e+01 -7.878522282935807652e+01 -5.501290217677853178e+01 -2.394284068288037659e+01 -1.103050127502735478e+01 -3.755152192246828236e+01 6.033684852049963183e+02 -9.395392940753350786e+01 -3.025445428925355031e+02 -5.228682497302794019e+01 -4.256639014281661559e+01
-8.464015232895986784e+00 -7.232357772928406803e+01 2.699207786153401400e-03 -5.838427532026519984e+00 -3.624287087290304576e+00 -3.068711738424425306e+01 -5.201014943558616466e+01 -6.920577485125070893e+01 -5.979159192076845386e+01 -2.308571009157111931e+01 -2.406815543820115266e+01 -1.765358006144679948e+01 -7.319781401411466604e+00 5.131270767400910593e+00 -2.892996455867175687e+00 -9.395392940753350786e+01 6.324159094472296374e+02 -9.964794411181482303e+01 -1.686311318054435304e+01 -3.796802682731019729e+00
-4.720242377171579307e+01 -2.360748751650241104e+02 -2.387952561158221698e+01 -3.654341382418284212e+01 -1.178361961368026201e+01 -1.146272540965963458e+02 -1.742594394834343348e+02 -2.302800743241008945e+02 -2.035858252808624229e+02 -9.359760020862401575e+01 -8.188646705342834764e+01 -5.807722266234809894e+01 -2.489110474117137528e+01 -2.450388263844354597e+00 -2.993301493728192142e+01 -3.025445428925355031e+02 -9.964794411181483724e+01 4.848867469850018210e+02 -5.528871829486565304e+01 -3.447916269287817670e+01
-2.010500328831743122e+00 -3.981837213986550950e+01 3.234786094523443456e+00 -8.476673766374314134e-01 -2.001688699949534467e+00 -1.493789498936761007e+01 -2.802683235808752471e+01 -3.750870956642506826e+01 -3.177950475474743186e+01 -1.024091542805382637e+01 -1.280096642802727303e+01 -9.657974108709190375e+00 -3.894854859286493109e+00 5.423744672240848885e+00 1.180548135923050523e+00 -5.228682497302794019e+01 -1.686311318054435304e+01 -5.528871829486565304e+01 5.557490303036809109e+02 8.989679926415519162e-01
1.687038472577143011e+02 -5.234068219751051743e+00 2.066292922789247086e+02 1.508270220168182618e+02 -6.652883713672430588e-01 1.228547503211815979e+02 3.507880422508249296e+01 3.289641686422240241e+01 6.847996448566266281e+01 1.561224022711316195e+02 2.701390207107819208e+01 2.643541898141094748e+00 8.104950072830565233e+00 1.664440052457680963e+02 1.770235011095859932e+02 -4.256639014281662980e+01 -3.796802682731012624e+00 -3.447916269287819091e+01 8.989679926415519162e-01 -5.552369262183564160e+02
