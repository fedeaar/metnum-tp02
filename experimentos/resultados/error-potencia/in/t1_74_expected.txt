-9.400000000000000000e+02
8.870000000000000000e+02
-8.160000000000000000e+02
8.070000000000000000e+02
-7.580000000000000000e+02
-7.460000000000000000e+02
6.990000000000000000e+02
-6.860000000000000000e+02
6.630000000000000000e+02
-6.520000000000000000e+02
6.330000000000000000e+02
5.650000000000000000e+02
5.170000000000000000e+02
-4.220000000000000000e+02
3.770000000000000000e+02
3.050000000000000000e+02
-2.580000000000000000e+02
1.750000000000000000e+02
1.710000000000000000e+02
1.650000000000000000e+02
